### PR TITLE
Modify CLI service commands to handle ingress info

### LINKED ISSUE #36: Extend CLI to enable ingress resource handling
**Story**
As a ML engineer, I would like to be able to see ingress information and use it to manage ingress, so that I have control over how my services are exposed beyond the k8s cluster.

**Tasks**
- Modify `cli.service_deployments.display_service_deployments` to include ingress information.
- Modify `cli.service_deployments.delete_service_deployment_in_namespace` to handle ingress alongside deployments.

### PR DESCRIPTION
**Pull request by AlexIoannides**
This PR incorporates ingress information into the CLI commands and closes #36

The CLI command impacted are:

### `bodywork service display`
I modified `display_service_deployments_in_namespace` that stands behind this CLI command, so that it prints better on the terminal - e.g.

```shell
----------------------------------------------
bodywork-serve-model-project--scoring-service:
----------------------------------------------
|- GIT_URL               https://github.com/bodywork-ml/bodywork-serve-model-project.git
|- GIT_BRANCH            bodywork-0.3.0-updates
|- REPLICAS_AVAILABLE    2
|- REPLICAS_UNAVAILABLE  0
|- EXPOSED_AS_SERVICE    true
|- CLUSTER_SERVICE_URL   http://bodywork-serve-model-project--scoring-service.test-ingress.svc.cluster.local
|- CLUSTER_SERVICE_PORT  5000
|- INGRESS_CREATED       true
|- INGRESS_ROUTE         /test-ingress/bodywork-serve-model-project--scoring-service
```

There's too much information to display if horizontally.

### `bodywork service delete`

Make sure that any ingress resources are deletes when deployments get killed.

### Misc

I also added a couple of one-line helper functions in `bodywork.k8s.service_deployments`:

- `ingress_route` - return the ingress route, given namespace and deployment name, using the bodywork convention.
- `cluster_service_url` - return the cluster service URL, given namespace and deployment name, using the bodywork convention.

Basically, the logic should be kept in one place, rather than assumed.

### CODE REVIEW

**Review by Marios85 — APPROVED** [review]
Everything looks good 😎
